### DISCUSSION TITLE
xplr version data

### DISCUSSION THREAD

**Discussion by auronandace** [sayanarijit/xplr#198] [Q&A] [answered]
Most programs include a --version flag to be able to identify what version the program is. It seems xplr doesn't have anything similar. Can a --version flag be added?

**Comment by sayanarijit** [comment, marked as answer]
I'm yet to decide on a CLI framework and implement the CLI args. But it's on the [TODO](https://github.com/sayanarijit/xplr/wiki/TODO) list.

For now, you have 2 options:

1. Debug using key `#` and grep the `version` field.
2. Drop into bash using key `:!` and `echo $XPLR_APP_VERSION`.
